### DISCUSSION TITLE
connectorX for AWS EC2 machines

### DISCUSSION THREAD

**Discussion by sebastian9610** [sfu-db/connector-x#711] [Q&A]
Hello.
I have a project with version 0.4.0 for Windows, for memory reasons we went to the AWS machines but when doing a pip install of connectorX it only installs version 0.2.3 and not the most recent one which generates conflicts with other libraries. I would like to know if anyone has a solution to this.
PS: apparently it is a fedora amazon linux.

**Comment by cowpowder** [comment]
Sounds like your ec2 is running on an Aarch64 linux architecture which connectorx doesn’t not support. You have two options here:
1) build connectorx from source in your machine/ docker image.
2) Provision an ec2 with amd64 archi instead.

On mobile right now hence unable to provide the links but you should be able to search past discussion/ issues by filtering for aarch64 or arm64. Also check if your ec2 is indeed aarch64!
